### PR TITLE
Cleaned up BundleServer CLI

### PR DESCRIPTION
**Pull request by breed**
Set the Log level to WARN if doing a CLI
Removed timestamps from logging if doing a CLI
Cleaned up some of the warnings
Don't try to be a gRPC server if doing a CLI

### CODE REVIEW

File: bundleserver/src/main/java/net/discdd/server/BundleServerApplication.java [hunk 1]
```diff
@@ -54,35 +54,41 @@ protected void refresh(ConfigurableApplicationContext applicationContext) {
...
         app.setBanner((e, s, o) -> o.println(BananaUtils.bananansi("DDD Bundle Server", Font.ANSI_SHADOW, Ansi.GREEN)));
 
         if (CommandProcessor.checkForCommand(args)) {
             // we are doing a CLI command, so don't start up like a server
             app.setWebApplicationType(WebApplicationType.NONE);
             app.setBannerMode(Banner.Mode.OFF);
             app.setLogStartupInfo(false);
+            System.setProperty("logging.level.root", "WARN");
+            System.setProperty("grpc.server.port", "-1");
+            System.setProperty("logging.pattern.console", "%-1level %msg%n");
```
Comment (JAReddy) [inline bundleserver/src/main/java/net/discdd/server/BundleServerApplication.java:89]: What is this property used for?

File: bundleserver/src/main/java/net/discdd/server/BundleServerApplication.java [hunk 2]
```diff
@@ -54,35 +54,41 @@ protected void refresh(ConfigurableApplicationContext applicationContext) {
...
+        }
+
         app.setBanner((e, s, o) -> o.println(BananaUtils.bananansi("DDD Bundle Server", Font.ANSI_SHADOW, Ansi.GREEN)));
 
         if (CommandProcessor.checkForCommand(args)) {
             // we are doing a CLI command, so don't start up like a server
             app.setWebApplicationType(WebApplicationType.NONE);
             app.setBannerMode(Banner.Mode.OFF);
             app.setLogStartupInfo(false);
+            System.setProperty("logging.level.root", "WARN");
```
Comment (JAReddy) [inline bundleserver/src/main/java/net/discdd/server/BundleServerApplication.java:87]: Will this System property override the default app properties or do we need to use Properties.put().

File: bundleserver/src/main/java/net/discdd/server/BundleServerApplication.java [hunk 3]
```diff
@@ -54,35 +54,41 @@ protected void refresh(ConfigurableApplicationContext applicationContext) {
...
+        try {
+            var properties = PropertiesLoaderUtils.loadProperties(resource);
+            app.setDefaultProperties(properties);
+            args = Arrays.copyOfRange(args, 1, args.length);
+        } catch (Exception e) {
+            logger.log(SEVERE, "Please enter valid properties file path!");
+            System.exit(1);
+        }
+
         app.setBanner((e, s, o) -> o.println(BananaUtils.bananansi("DDD Bundle Server", Font.ANSI_SHADOW, Ansi.GREEN)));
```
Comment (JAReddy) [inline bundleserver/src/main/java/net/discdd/server/BundleServerApplication.java:80]: Do we need the banner for CLI?

File: bundleserver/src/main/java/net/discdd/server/BundleServerApplication.java (hunk 1 above)
Comment (breed) [reply bundleserver/src/main/java/net/discdd/server/BundleServerApplication.java:89]: that is to remove the timestamp and component field from the log. it makes it more readable on the screen.

File: bundleserver/src/main/java/net/discdd/server/BundleServerApplication.java (hunk 2 above)
Comment (breed) [reply bundleserver/src/main/java/net/discdd/server/BundleServerApplication.java:87]: yes, system properties trump even the application.yml properties. there is a nice reference about the precedence: https://docs.spring.io/spring-boot/docs/1.5.6.RELEASE/reference/html/boot-features-external-config.html it's a bit old but seems valid still.

File: bundleserver/src/main/java/net/discdd/server/BundleServerApplication.java (hunk 3 above)
Comment (breed) [reply bundleserver/src/main/java/net/discdd/server/BundleServerApplication.java:80]: that gets ignored because banner mode is set to off.

**Review by JAReddy — APPROVED** [review]

**Pull request merged**